create/serviceconnection.go

```diff
@@ -149,19 +193,19 @@ func (cmd *serviceConnectionCmd) newServiceConnection(namespace string) (*networ
 	return sc, nil
 }
 
-func groupKindFromKind(kind string) (metav1.GroupKind, error) {
+func groupVersionKindFromKind(kind string) (schema.GroupVersionKind, error) {
```

Comment:
Do we maybe want to check the `group` as well? Just if we will ever have the same `Kind` in multiple groups?